qt/widgets/common/src/MantidWSIndexDialog.cpp

```diff
@@ -457,7 +457,7 @@ void MantidWSIndexWidget::initWorkspaceBox() {
   m_wsBox = new QVBoxLayout;
   const QString wsIndices = m_wsIndexIntervals.toQString();
   const QString label = "Enter Workspace Indices: " + wsIndices;
-  m_wsMessage = new QLabel(tr(label.toAscii().constData()));
+  m_wsMessage = new QLabel(tr(label.toLatin1().constData()));
```

Comment:
I think `label.toLatin1().constData()` should be replaced with `qPrintable(label)`

http://doc.qt.io/qt-5/qtglobal.html#qPrintable

Reply:
Yep, that would be clearer. This was just a search-and-replace. I'll update these in a separate PR.